src/boot/efi/boot.c

```diff
@@ -2041,8 +2043,8 @@ static EFI_STATUS boot_windows_bitlocker(void) {
 
                 if (streq16((char16_t *) (buf + offset), L"Windows Boot Manager")) {
                         err = efivar_set_raw(
-                                EFI_GLOBAL_GUID,
-                                L"BootNext",
+                                MAKE_GUID_PTR(EFI_GLOBAL_VARIABLE),
+                                u"BootNext",
```

Comment:
This seems unrelated to this commit. Should probably be somewhere else or its own commit

Reply:
It's porting to the new way of using GUIDs, so I think it's relevant. `EFI_GLOBAL_GUID` uses the old style and a non-standard (compared to the spec) name for this var. Might as well do both at once…
Whether these are all touched in one commit or split out makes little difference imho. They're gonna be commit noise anyways.